### ISSUE TITLE
[CoE Starter Kit - BUG] Help migrating CoE to new environment - until we can directly migrate with Dataflows. 

### ISSUE THREAD

**Issue by Jenefer-Monroe** [microsoft/coe-starter-kit#1507]
### Describe the issue

Because of a product limitaiton (which is being worked on) in Data Flows in which you cannot preserve GUIDs, you cannot directly migrate the data in a CoE from one envt to another. 

For now, this bug can be looked at for help in figuring out how to migrate. Ideas of what people have worked on will be below. 

Note this is not intended to be supported advice, just a place for the community to work together to resolve. 

Why would you want to do this? 
- Upgrade bugs are preventing you from getting new versions of the starter kit
- You want to boot a new dev environment for CoE with your existing content
- You want a fresh install in order to have ownership of objects and connections to be clean
- etc

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.18

### What app or flow are you having the issue with?

migration

**Comment by Jenefer-Monroe** [comment]
### Just Inventory and Audit Logs

Here's one technique used by @osbornecc in #1190 :
Assumption: They only used CoE for inventory. No other augmentable fields were used. So they just needed to not lose their audit logs

Steps:
- Create new envt
- Setup core again and turn on all the inventory related flows [Setup Core](https://docs.microsoft.com/power-platform/guidance/coe/setup-core-components)
- Wait for inventory to populate
- Migrate data from old CoE **_Audit Log_** table in Core to new CoE environment in one of the supported techniques.
> Note this table does not need GUIDs preserved and hence is not subject to the product bug mentioned. There are many techniques to migrate such data including:
> [dataverse migration with dataflows](https://docs.microsoft.com/powerapps/developer/data)
> [By exporting from old into Excel](https://docs.microsoft.com/powerapps/maker/data-platform/data-platform-import-export#:~:text=Select%20Get%20Data.,Select%20Get%20data%20from%20Excel)
- At this point you will have all the data except the **_App Last Launched On_** field of the **_PowerApps App_** table. 
- The technique we used to do this in 1190 was to import and run this flow in the new CoE environment
> [rehydrateapplastlaunchedfield_20211118171220.zip](https://github.com/microsoft/coe-starter-kit/files/7564657/rehydrateapplastlaunchedfield_20211118171220.zip)
> Note this will take some time, my test tenant with ~1350 apps took 15 minutes
- Lastly install and set up audit logs in the new envionment: [setup audit logs](https://docs.microsoft.com/power-platform/guidance/coe/setup-auditlog)

**Comment by Jenefer-Monroe** [comment]
Found that we dont need to differentiate between custom and not here so dont need the verbose info. Able to always behave the same now custom/not verbose/small. 

LF and running a validation suite, will send to partner to test

**Comment by DaveTheTeamsGuy** [comment]
Hi - I'm attempting an audit log import from an old CoE instance to new CoE instance.  I have exported the old audit log table to excel, and when I attempt to import it in the new instance (Data > Get Data > Get Data from Excel), everything maps OK, but when I run the import, it runs for a while then I get the following message:  "Import failed: There was an error when getting the project execution status. Executing"

I've tried twice, same message both times.  I checked dataflows (as suggested at [this link](https://powerusers.microsoft.com/t5/Microsoft-Dataverse/Import-failed-There-was-an-error-when-getting-the-project/td-p/742590) thinking I might see an error, but there's no dataflows listed.  Also, I'm guessing since I did this twice, I just doubled up the data in the table.  Thoughts on how to proceed?
